### PR TITLE
Adds datatable to all teams page [SCI-936]

### PR DESCRIPTION
**Pull request by okriuchykhin**
https://biosistemika.atlassian.net/browse/SCI-936

### CODE REVIEW

File: app/datatables/teams_datatable.rb [hunk 1]
```diff
@@ -0,0 +1,106 @@
+class TeamsDatatable < AjaxDatatablesRails::Base
+  include InputSanitizeHelper
+
+  def_delegator :@view, :link_to
+  def_delegator :@view, :team_path
+  def_delegator :@view, :leave_user_team_html_path
+
+  MEMEBERS_SORT_COL = 'members'.freeze
```
Comment (ZmagoD) [inline app/datatables/teams_datatable.rb]: Can you put this constant in constants.rb?

File: app/views/users/settings/teams.html.erb [hunk 2]
```diff
@@ -13,85 +13,37 @@
...
+        <%= link_to new_team_path, class: "btn btn-default", style: "margin-left: 30px;" do %>
+          <span class="glyphicon glyphicon-plus"></span>
+          <span class="hidden-xs">
+            <%= t("users.settings.teams.index.new_team") %>
+          </span>
+        <% end %>
+      </span>
     </div>
 
     <% if @member_of > 0 %>
```
Comment (ZmagoD) [inline app/views/users/settings/teams.html.erb]: more ruby like way to put it :smile_cat:  @member_of.any?

File: app/assets/javascripts/users/settings/teams_datatable.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    },
+    colReorder: {
+      fixedColumnsLeft: 1000000 // Disable reordering
+    },
+    columnDefs: [{
+      targets: [0, 1],
+      orderable: true,
+      searchable: false
+    }, {
+      targets: [2],
```
Comment (ZmagoD) [inline app/assets/javascripts/users/settings/teams_datatable.js]: I think you don't need to put this in an array.

File: app/assets/javascripts/users/settings/teams_datatable.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+      searchable: false,
+      orderable: true
+    }, {
+      targets: [3],
```
Comment (ZmagoD) [inline app/assets/javascripts/users/settings/teams_datatable.js]: same here

File: app/datatables/teams_datatable.rb (hunk 1 above)
Comment (okriuchykhin) [inline app/datatables/teams_datatable.rb]: It's only used in this datatable, so is it really worth to move it to constants.rb?

File: app/views/users/settings/teams.html.erb (hunk 2 above)
Comment (okriuchykhin) [inline app/views/users/settings/teams.html.erb]: Not my code, but i can fix it)

Comment (okriuchykhin) [inline app/views/users/settings/teams.html.erb]: And @member_of is just a number? Can we use .any? here?

**Comment by ZmagoD** [comment]
:rocket:

**Pull request merged**